examples/boto3/service-2.sdk-extras.json

```diff
@@ -26,6 +37,34 @@
             }
         },
         "AllowUnordered":{"type":"boolean"},
+        "PutObjectRequest": {
```

Comment:
could you please add doc URL: https://docs.ceph.com/docs/master/radosgw/s3/objectops/#append-object